### ISSUE TITLE
Doubt regading prediction with VectorMapNet model

### ISSUE THREAD

**Issue by quangnhat185** [Mrmoore98/VectorMapNet_code#12]
I crop this snippet in `dghead.py` which is used to perform prediction. It checked and see that `batch` seems to contain the ground-truth detection (class labels and bounding boxes in `batch['det']`). Since VectorMapNet supposes to be an end-to-end map learning model, I don't understand why ground truth is require to perform inference here . Did I understand something wrong here? 

I want to use your model make prediction with my custom camera data, can you suggest on how to do it?

```python
    @torch.no_grad()
    def inference(self, batch: dict, context: dict, gt_condition=False, **kwargs):
        '''
            num_samples_batch: number of sample per batch (batch size)
        '''
        outs = {}
        bbox_dict = self.det_net(batch['det'], context=context)
        bbox_dict = self.det_net.post_process(bbox_dict)
        
```
